github-for-jira.sd.yml

```diff
@@ -218,7 +218,7 @@ config:
 
     GLOBAL_AGENT_NO_PROXY: cryptor # needed because of proxy.ts
     CRYPTOR_URL: http://cryptor:26272
-    CRYPTOR_SIDECAR_ENCRYPTION_KEY_ALIASES: micros/github-for-jira/github-server-app-secrets
+    CRYPTOR_SIDECAR_ENCRYPTION_KEY_ALIASES: micros/github-for-jira/github-server-app-secrets,micros/github-for-jira/jira-instance-secrets
```

Comment:
do we really need to specify each key that we want to use?  Is there a way to do wildcards?

Reply:
I can double check again. But in the doc, it just say `comma separated blah blah`

Reply:
There's no wildcard